pulp_2to3_migration/app/plugin/content.py

```diff
@@ -320,30 +322,35 @@ def get_remote_by_importer_id(importer_id):
                             artifact = Artifact()
 
                         lces = pulp2lazycatalog.filter(pulp2_storage_path=image_path)
-                        if lces:
-                            remote_declarative_artifacts = []
 
+                        if not lces and not downloaded:
+                            continue
+
+                        if lces:
```

Comment:
this condition is unnecessary

Reply:
Could you elaborate?

Reply:
You can just remove the if condition and keep only the for loop. the `lces` from line 324 will wether be an empty queryset or will contain some data to loop trough.